src/traces/bar/layout_attributes.js

```diff
@@ -51,5 +51,19 @@ module.exports = {
             'Sets the gap (in plot fraction) between bars of',
             'the same location coordinate.'
         ].join(' ')
-    }
+    },
+    barshowtotal: {
+        valType: 'boolean',
+        dflt: false,
+        role: 'style',
+        editType: 'plot',
+        description: 'Display the total value of each bar.'
+    },
+    totaltemplate: {
```

Comment:
We may want this attribute or a similar one to control hover.
It may be better to have 
```js
stackbartotal: {
  texttemplate: ...,
  hovertemplate: ...
}
```
And there is no need for `barshowtotal` as it could be enabled by `texttemplate`.